nltk/parse/stanford.py

```diff
@@ -125,25 +125,84 @@ def raw_parse(self, sentence, verbose=False):
         :type sentence: str
         :rtype: iter(Tree)
         """
-        return next(self.raw_parse_sents([sentence], verbose))
+        if properties is None:
```

Comment:
I don't really like how the properties are treated here. The idea is to let user to define additional properties in the most transparent and similar way to the original corenlp docs.